### PR TITLE
Test

### PR DESCRIPTION
**Pull request by khanfaizan1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# [Scalania](http://scalania.pl) - learn Scala by examples
+# [Scalania] (http://scalania.pl) - Learn Scala by Examples,,,,,,,,,,,
```
Comment (jaceklaskowski) [inline README.md]: Why so many commas?

File: dsfasfd [hunk 2]
```diff
@@ -0,0 +1,30 @@
+node {
```
Comment (jaceklaskowski) [inline dsfasfd:1]: Is the file valid? The file name suggests the opposite.

**Review by jaceklaskowski — CHANGES_REQUESTED** [review]
1. Does `JenkinsFile` have to be in the main directory?

**Comment by khanfaizan1** [comment]
Sorry I am new to Github and was just testing Jenkins Webhooks,

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# [Scalania] (http://scalania.pl) - Learn Scala by Examples,,,,,,,,,,,
+# [Scalania] (http://scalania.pl) - Learn Scala by Examples,,
```
Comment (jaceklaskowski) [inline README.md]: Why two commas at the end?

**Review by jaceklaskowski — CHANGES_REQUESTED** [review]

**Comment by khanfaizan1** [comment]
Can one of the admins verify this patch?

**Comment by jaceklaskowski** [comment]
Closing the pull request as it was inactive for some time.

**Pull request closed without merging**